src/coreclr-embedding-test/src/main.cpp

```diff
@@ -1,167 +1,49 @@
-#include <cstdint>
-#include <cstddef>
-#include <coreclr/coreclr_delegates.h>
-#include <coreclr/hostfxr.h>
-#include <unistd.h>
-#include <dlfcn.h>
-#include <cstdio>
-#include <filesystem>
+#define DEBUG 1
```

Comment:
We could use something like NDEBUG, right?
Or a compile time definition through CMake

Reply:
Placed that there because my Mac was not adding the definitions through Cmake, but yeah, absolutely this should be delegated to an external project file such as our Cmake script